src/main/logutils.ts

```diff
@@ -818,6 +818,24 @@ const watchLogs = (logdir: any) => {
     return [name, realm];
 }
 
+/**
```

Comment:
Feels like this should be in recorder.ts, but I think it's a bit tricky because of the metadata being a variable in logutils so not proposing you do anything about it now. Something to keep in mind for us in the future.

Reply:
There's actually a lot of things I wanted to move around but due to where some data is located (like `metadata`) it isn't feasible for now - _rest assured_ I will restructure more stuff along the way ;)